src/templates/base/2019/base.html

```diff
@@ -341,18 +347,37 @@
 {% endmacro %}
 
 {% macro year_switcher(switcher_name) %}
+{# Only show if we currently support multi-year #}
+{% if year_live('2020') %}
 <div class="year-switcher">
   <label for="year-switcher-{{switcher_name}}" class="visually-hidden">{{ self.year_switcher() }}</label>
   <select id="year-switcher-{{switcher_name}}">
       {% for y in supported_years %}
         <option
           {% if y == year %}selected="selected"{% endif %}
-          value="{{ url_for('table_of_contents', **get_view_args(year=y)) }}">
-          {{ edition(y) }}
+          value="{{ url_for(request.endpoint, **get_view_args(lang=lang,year=y)) }}">
+          {{ y }}
         </option>
       {% endfor %}
+      {# If the current page doesn't support a year, then show a home link for that year #}
```

Comment:
After demoing this, I'm torn on this behavior. It can create different experiences on different chapters. I might prefer the year switchers to always navigate to the year's home page. WDYT?

Reply:
I disagree. If I'm into SEO (for example) I might want to flick between the different years and read chapters on each. Flicking back to home page creates a needless extra step.

At the end of the day, these are supposed to be short term workarounds. We should aim to launch the English version with most (if not all) the chapters at which point there is no different experience.

In that way it's similar with the translations where we flick back to English versions. I know you had similar reservations about that though. And yes the translations may take longer (and we've seen with 2019 that there is a risk that some will be in an uncompleted state for the long term), but that is unfortunate and I'd rather not hold back the languages that are fully available because some are not.